src/layouts/DashboardLayout.vue

```diff
@@ -178,6 +178,17 @@
               class="float-right mt-1"
             ></b-spinner>
           </b-alert>
+          <b-alert
+            class="mt-4 mb-0"
+            variant="warning"
+            :show="highMemoryUsage"
+            dismissible
+            @dismissed="clearMemoryWarning()"
+          >
+            <b-icon icon="exclamation-circle" class="mr-2"></b-icon>
+            Your Umbrel is struggling to run all of your apps. Consider
+            uninstalling some apps or upgrading your Umbrel.
```

Comment:
```suggestion
            uninstalling some apps or upgrading your Umbrel's hardware.
```

Upgrading your Umbrel sounds a bit like a system update in my opinion.

Reply:
Yes, great point.